### ISSUE TITLE
use texture compression where it doesn't hurt

### ISSUE THREAD

**Issue by RandomEtc** [cooperhewitt/Planetary#159]
Ryan has a pull request for Cinder which allows it to load textures that have been compressed for iOS (powervr). This is a big performance/memory recommendation in the powervr guides.

Let's take a look: https://github.com/cinder/Cinder/pull/35

(Apparently it screws with the edges of textures so it's even more important to build texture atlases).

**Comment by RandomEtc** [comment]
OK, Jesper made good progress on this front. He says "the tool comes with theiOS DSK post 4.3 /Developer/Platforms/iPhoneOS.platform/Developer/usr/bin/texturetool"

The full command line to test with `galaxy.pvr` in the "compressed" branch was:
`/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin/texturetool -e PVRTC --channel-weighting-linear --bits-per-pixel-4 -o galaxy.pvr galaxy.jpg`

We need to merge Ryan's branch of Cinder (for compressed textures) in with our own (for task switching fixes) before we can test this with all the images, but after a little bit more testing with Instruments (for framerate) I'd like to use it for everything we can.

**Comment by RandomEtc** [comment]
Ryan's compression branch of Cinder is merged with ours now so this should be OK to enable if we want.

For square textures we should start using .pvr images wherever we can. Especially things that aren't drawn smaller - right now we can't enable mip-mapping for compressed textures, though Ryan said he was working on it. This is why I turned it off for the galaxy. Other things didn't look right when I scaled them to squares, but as I say the UI is a good bet for this now the textures are square.

This is about saving GPU memory and bandwidth, it will make the app a slightly bigger download but it will also avoid needing to decompress PNGs which might save us a bit of startup time and should help avoid memory warnings.
